lib/fixture/locator/helpers.ts

```diff
@@ -42,13 +50,61 @@ const buildTestingLibraryScript = async ({config}: {config: Config}) => {
 const synchronousQueryNames = allQueryNames.filter(isNotFindQuery)
 
 const queriesFor = (pageOrLocator: Page | Locator) =>
-  synchronousQueryNames.reduce(
-    (rest, query) => ({
+  allQueryNames.reduce((rest, query) => {
+    if (isFindQuery(query)) {
+      return {
+        ...rest,
+        [query]: async (
```

Comment:
IMHO, this should be hoisted out into a function named something like `createFindQuery`.

Reply:
Heh, totally agree.